### ISSUE TITLE
Add other programs

### ISSUE THREAD

**Issue by UltraStudioLTD** [netxs-group/vtm#168]
I saw office-like applications in your Twits and YouTube videos. Will you add them too or not? Also if you don't mind can you share their code?

**Comment by o-sdn-o** [comment]
Yes, these programs are part of vtm.  At the moment, these are just drafts for the purpose of demonstrating the capabilities of the Desktopio framework. 

These programs are enabled when the DEMO macro is defined in the vtm.cpp file.

They are located at ‘vtm/src/netxs/apps/’ in the source code tree.

**Comment by UltraStudioLTD** [comment]
Good work with **VTM**. You are making what would be considered impossible and you should be proud for it. What will you do after VTM? Add ImTUI (ImGUI for Terminal) support? It will greatly help VTM

**Comment by UltraStudioLTD** [comment]
> * term.hpp - Terminal Emulator

Hmm... Terminal Emulator in Terminal Emulator... 🤔
I should test if it will run **VTM** again

**Comment by UltraStudioLTD** [comment]
I tested it and on Termux it doesn't run. It does some kinda infinitely recursive VTM view. Don't know about other platforms

**Comment by UltraStudioLTD** [comment]
Do you use **pty** (*pseudo-terminal*) for **Term** app?

**Comment by o-sdn-o** [comment]
> Do you use pty (pseudo-terminal) for Term app?

Inside Term there is a platform-specific PTY object.  For POSIX it's PTY, for Windows it's ConPTY.

**Comment by o-sdn-o** [comment]
> Emulator in Terminal Emulator... 🤔I should test if it will run VTM again

You can try running vtm inside vtm as a different user.

**Comment by o-sdn-o** [comment]
> What will you do after VTM?

**Comment by o-sdn-o** [comment]
vtm is a multi-user environment in the sense that it is possible to connect any number of participants at the same time.  A session is created for the current user account, allowing any number of clients on behalf of this account. Just run vtm in the second OS window to test it.

**Comment by UltraStudioLTD** [comment]
Closing in favor of #183
